heron/common/src/cpp/config/heron-internals-config-vars.h

```diff
@@ -152,6 +152,9 @@ class HeronInternalsConfigVars {
   // The maximum packet size in MB of stream manager's network options
   static const sp_string HERON_STREAMMGR_NETWORK_OPTIONS_MAXIMUM_PACKET_MB;
 
+  // The interfaces to bind to as a comma separated list
+  static const sp_string HERON_STREAMMGR_NETWORK_OPTIONS_BIND_INTERFACE;
```

Comment:
If you want to allow multiple interfaces, please rename it to INTERFACES (singular vs plural). And I am not sure how much sense it makes to support multiple interfaces here, could you explain?